spyderlib/spyder.py

```diff
@@ -1718,17 +1669,17 @@ def setup_default_layouts(self, index, settings):
         self._custom_layout_timer.start(100)

```

Comment:
Maybe we could avoid using this timer on a first start? I'd be happy with that temporary solution, at least for beta1 :-)